### PR TITLE
adding VS Code code tour

### PR DESCRIPTION
**Pull request by pchittum**
### What does this PR do?
Adds a code tour for VS Code to the sample app.

### CODE REVIEW

File: .tours/welcome-to-lightning-web-components-recipes.tour [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    },
+    {
+      "file": "force-app/main/default/lwc/helloBinding/helloBinding.html",
+      "line": 7,
+      "description": "The template for an LWC is contained in a `.html` file that shares a name with the JavaScript module file. \n\nProperties and functions from the JavaScript are referenced in the template with `{brackets}`, like you see here with `{greeting}` and `{handleChange}`. "
+    },
+    {
+      "file": "force-app/main/default/lwc/helloBinding/helloBinding.js-meta.xml",
+      "line": 2,
+      "description": "All the files of a LWC are contained in a single directory. Together they are called a component bundle. \n\nIn addition to the `.js` and `.html` files, this can also include CSS, submodules, alternate templates, and more. \n\nThis `.xml` file here defines how the LWC is used by the Salesforce platform. \n\nFor more details on the structure of an LWC, check out the LWC developer [guide](https://developer.salesforce.com/docs/component-library/documentation/en/lwc). ",
```
Comment (albarivas) [inline .tours/welcome-to-lightning-web-components-recipes.tour:22]: In this sentence you say first "a LWC" and then "an LWC". I guess the last one is incorrect.

File: .tours/welcome-to-lightning-web-components-recipes.tour [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      "selection": {
```
Comment (albarivas) [inline .tours/welcome-to-lightning-web-components-recipes.tour:23]: You are selecting the whole line, is this intentional? I think this is equivalent to just indicate the line number. I think selection is more suitable for when you want to highlight a specific word or similar.

File: .tours/welcome-to-lightning-web-components-recipes.tour [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
...this is to use our [Quick Start](https://trailhead.salesforce.com/en/content/learn/projects/quick-start-lwc-recipes-app) on Trailhead. This will walk you through deploying the app and getting to know it a bit, as well as explore some of the other code samples. \n- But if you prefer to take yourself through the app, you can skip right to the installation instructions that are available in the git repo's [README](https://github.com/trailheadapps/lwc-recipes#installing-the-app-using-a-scratch-org) file. From there you can explore the exmaples bit by bit. There is also an in-app tour that will familiarize you with the broad categories of recipes available. \n- If you're interested in using LWC independent of the Salesforce platform, be sure to check out the [lwc.dev](lwc.dev) website and the open source LWC Recipes OSS [sample app](https://github.com/trailheadapps/lwc-recipes-oss).\n- Lastly, if you've looked around and don't see something you think should be a recipe, please open an [issue](https://github.com/trailheadapps/lwc-recipes/issues/new?template=Feature_Request.md) and let us know your idea. We'd love to discuss it with you. \n\nThanks! And enjoy the LWC Recipes sample app. "
```
Comment (albarivas) [inline .tours/welcome-to-lightning-web-components-recipes.tour:42]: The lwc.dev link is broken. I think you must add the https:// before.

**Review by albarivas — APPROVED** [review]
I think the approach that you taken is great. It gives an idea of how the app is structured, the kind of recipes tha it includes (from the easiest one to more complex examples) and where to look for more. I left a couple of minor comments.

**Review by pozil — APPROVED** [review]
Looking good. I agree that the CodeTour UI could be improved.

File: .tours/welcome-to-lightning-web-components-recipes.tour (hunk 1 above)
Comment (pchittum) [reply .tours/welcome-to-lightning-web-components-recipes.tour:22]: thanks. good catch.

File: .tours/welcome-to-lightning-web-components-recipes.tour (hunk 2 above)
Comment (pchittum) [reply .tours/welcome-to-lightning-web-components-recipes.tour:23]: yikes. I didn't know I did that. I'll look at this.

**Pull request merged**